Modules/@babylonjs/react-native-windows/windows/BabylonReactNative/EngineView.cpp

```diff
@@ -183,7 +183,7 @@ namespace winrt::BabylonReactNative::implementation {
             if (propertyName == "isTransparent")
             {
                 bool isTransparent = propertyValue.AsBoolean();
-                this->CompositeMode(isTransparent ? ElementCompositeMode::MinBlend : ElementCompositeMode::Inherit);
+                BabylonNative::UpdateAlphaPremultiplied(isTransparent);

```

Comment:
So on Windows, when the isTransparent flag is true, we need to set the platform agnostic alpha premultiplied to true as well. Is this only for Windows? Will the logic be different for the other platforms (iOS and Android)?